pyproject.toml

```diff
@@ -7,6 +7,9 @@ description = "Open Source HR & Payroll Software"
 requires-python = ">=3.10"
 readme = "README.md"
 dynamic = ["version"]
+dependencies = [
+    "frappe-client~=0.1.0.dev"
```

Comment:
We don't need to add this as a dependency. 
Frappe has frappeclient:
`from frappe.frappeclient import FrappeClient`

https://github.com/frappe/frappe/blob/develop/frappe/frappeclient.py